### PR TITLE
Antac companies

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/antac.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    def find_pep_companies(self, pep_name):
+        # TODO: retreive a public person object from another DB
+        pep = pep_name
+        self.nodes.append(pep)
+        founder_of = FounderFull.objects.filter(name=pep_name)
+        if not founder_of:
+            return
+        for founder in founder_of:
+            self.nodes.append(founder.company)
+            self.links.append(Link(pep, founder.company))
```
Comment (IegorSoboliev) [inline business_register/converter/antac.py:29]: @roman-tiukh we can return pep.id, founder.company.id here

Comment (roman-tiukh) [reply business_register/converter/antac.py:29]: it's question?

File: business_register/converter/antac.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class Link:
+
+    def __init__(self, source, target):
+        self.source = source
+        self.target = target
+
+
+linked_companies = []
```
[the remark this answers is not in the document]
Comment (IegorSoboliev) [reply business_register/converter/antac.py:11]: sorry, previous decision. Deleted)

File: business_register/converter/antac.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/antac.py:29]: just opportunity)

Comment (roman-tiukh) [reply business_register/converter/antac.py:29]: it is, in links it is not necessary to lay down the entire object, use only id.

File: business_register/converter/antac.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    def find_pep_companies(self, pep_name):
+        # TODO: retreive a public person object from another DB
+        pep = pep_name
+        self.nodes.append(pep)
+        founder_of = FounderFull.objects.filter(name=pep_name)
+        if not founder_of:
+            return
+        for founder in founder_of:
+            self.nodes.append(founder.company)
+            self.links.append(Link(pep, founder.company))
```
Comment (roman-tiukh) [inline business_register/converter/antac.py:26]: > we can return pep.id, founder.company.id here

do it

Comment (IegorSoboliev) [reply business_register/converter/antac.py]: done

**Pull request merged**